src/pages/home/Home.scss

```diff
@@ -41,7 +41,7 @@ $distance-height-variation-after: 40px;
       top: 0;
       height: 100%;
       width: 100%;
-      z-index: -2;
+      z-index: index(behind_background);
```

Comment:
We could do something like `z-index: index(background) - 1;`, this avoids creating a new z-index "category".

Reply:
Gotcha :).